### PR TITLE
fix(guide): added hints & solutions for React and Redux challenge related Guide articles

### PR DESCRIPTION
**Pull request by RandellDawson**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

This PR was created based on a branch of user @ShepherdNg who created PR #35781.  That PR had an extra file on it which did not need to be on the PR and a few of the solutions were only partial solutions and some stub info was left in the articles. The user was asked to fix the issues, but the PR went stale.  We attempted to fix the issues on user's behalf, but the user did not give maintainers write access to the user's branch, so I fixed the issues and submitted this PR using the user's branch.

### CODE REVIEW

File: guide/english/certifications/front-end-libraries/react-and-redux/moving-forward-from-here/index.md [hunk 1]
```diff
@@ -3,8 +3,7 @@ title: Moving Forward From Here
 ---
 ## Moving Forward From Here
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/certifications/front-end-libraries/react-and-redux/moving-forward-from-here/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
-
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
-
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+### Solution
```
Comment (thecodingaviator) [inline guide/english/certifications/front-end-libraries/react-and-redux/moving-forward-from-here/index.md]: Please change this one to the details and summary thing you've done for all challenges

**Review by thecodingaviator — CHANGES_REQUESTED** [review]

File: guide/english/certifications/front-end-libraries/react-and-redux/moving-forward-from-here/index.md (hunk 1 above)
Comment (RandellDawson) [reply guide/english/certifications/front-end-libraries/react-and-redux/moving-forward-from-here/index.md]: I just pushed this change up.

**Review by thecodingaviator — APPROVED** [review]

**Review by thecodingaviator — APPROVED** [review]
LGTM

**Comment by RandellDawson** [comment]
@thecodingaviator Is this one good to be merged?

**Comment by thecodingaviator** [comment]
The formatting and all looks fine to me. I’ll have another look at then Code later today

**Review by thecodingaviator — APPROVED** [review]
LGTM

**Pull request merged**